content_script.js

```diff
@@ -71,11 +72,10 @@
 
   const reportFailure = (status) => {
     const statusCodes = {
-      1: 'INVALID_SELECTION: The selection provided could not be used.',
-
-      2: 'AMBIGUOUS: No unique fragment could be identified for this selection.',
-      3: 'TIMEOUT: Computation could not complete in time.',
-      4: 'EXECUTION_FAILED: Unknown error.',
+      1: 'INVALID_SELECTION: ❌ The selected text is too short or does not contain enough unique words. Please choose a longer or more specific phrase.',
+      2: 'AMBIGUOUS: ❌ The selected text appears multiple times on this page. Please select a more unique phrase to create an accurate link.',
+      3: 'TIMEOUT: ⏳ The process took too long. This may be due to a large page size or slow browser performance. Try selecting a shorter text segment or reloading the page.',
```

Comment:
```suggestion
      3: 'TIMEOUT: ⏳ The process took too long. This may be due to a large page size or slow browser performance. Try selecting a different text segment.',
```